src/features/bonification/pages/BonificationBirthPlace.native.test.tsx

```diff
@@ -59,16 +53,28 @@ describe('BonificationBirthPlace', () => {
 
       render(<BonificationBirthPlace />)
 
-      const countryOfBirthField = screen.getByTestId('Entrée pour le pays de naissance')
-      await userEvent.type(countryOfBirthField, 'France')
-      const cityOfBirthField = screen.getByTestId('Entrée pour la ville de naissance')
-      await userEvent.type(cityOfBirthField, 'Toulouse')
-
-      const button = screen.getByText('Continuer')
-      await userEvent.press(button)
+      await completeFormAndContinue()
 
       expect(setBirthCountrySpy).toHaveBeenCalledWith('France')
       expect(setBirthCitySpy).toHaveBeenCalledWith('Toulouse')
     })
   })
 })
+
+async function completeFormAndContinue() {
+  // Fill the country input
+  const countryOfBirthField = screen.getByTestId('Entrée pour le pays de naissance')
+  await userEvent.type(countryOfBirthField, 'France')
+
+  // Select the suggested country
+  const countrySuggestion = screen.getByText('France')
+  await userEvent.press(countrySuggestion)
+
+  // Fill in the city input
+  const cityOfBirthField = screen.getByTestId('Entrée pour la ville de naissance')
+  await userEvent.type(cityOfBirthField, 'Toulouse')
+
+  // Attempt to go validate the form
+  const button = screen.getByText('Continuer')
+  await userEvent.press(button)
+}
```

Comment:
nitpick: pour plus de lisibilité j'aime bien séparer les 3 phases du tests

```suggestion
async function completeForm() {
  // Fill the country input
  const countryOfBirthField = screen.getByTestId('Entrée pour le pays de naissance')
  await userEvent.type(countryOfBirthField, 'France')

  // Select the suggested country
  const countrySuggestion = screen.getByText('France')
  await userEvent.press(countrySuggestion)

  // Fill in the city input
  const cityOfBirthField = screen.getByTestId('Entrée pour la ville de naissance')
  await userEvent.type(cityOfBirthField, 'Toulouse')
}

async function continue() {
  // Attempt to go validate the form
  const button = screen.getByText('Continuer')
  await userEvent.press(button)
}
```